angular/projects/spark-angular/src/lib/components/sprk-icon/sprk-icon.component.spec.ts

```diff
@@ -31,17 +31,52 @@ describe('SprkIconComponent', () => {
     expect(component.getClasses()).toEqual('sprk-c-Icon');
   });
 
+  it('should add the correct icon based on the icon type set', () => {
+    component.iconName = 'bell';
+    fixture.detectChanges();
+    expect(
+      iconElement.querySelector('use').hasAttribute('xlink:href'),
+    ).toBeTruthy();
+    expect(iconElement.querySelector('use').getAttribute('xlink:href')).toEqual(
+      '#bell',
+    );
+  });
+
+  // TODO: deprecate iconType
   it('should add the correct icon based on the icon type set', () => {
     component.iconType = 'bell';
     fixture.detectChanges();
     expect(
-      iconElement.querySelector('use').hasAttribute('xlink:href')
+      iconElement.querySelector('use').hasAttribute('xlink:href'),
     ).toBeTruthy();
     expect(iconElement.querySelector('use').getAttribute('xlink:href')).toEqual(
-      '#bell'
+      '#bell',
     );
   });
 
+  it('should add the default viewBox if not set', () => {
```

Comment:
We should also update these tests to use `iconName`

Reply:
✅ Good catch! fixed this all around the test file and pushed up